Mathlib/LinearAlgebra/Matrix/Determinant/TotallyUnimodular.lean

```diff
@@ -155,8 +150,6 @@ lemma fromRows_one_isTotallyUnimodular_iff [DecidableEq n] (A : Matrix m n R) :
   fromRows_isTotallyUnimodular_iff_rows <| fun i ↦ Or.inr
     ⟨i, funext fun j ↦ by simp [one_apply, Function.update_apply, eq_comm]⟩
 
-alias ⟨_, IsTotallyUnimodular.fromRows_one⟩ := fromRows_one_isTotallyUnimodular_iff
```

Comment:
Let's keep this alias since it gives dot notation

Reply:
OK, I'll put it back.